### ISSUE TITLE
Which version to use?

### ISSUE THREAD

**Issue by keppo070** [clara-j/media_cleaner#30]
I'm interested in your script. It seems to do what I what I was searching for. 

I'm a bit confused about this script or a similar script from https://github.com/terrelsa13/media_cleaner. They seem to be the same but the read.me is completly differrent and the last edited seem also different.

Shoud I try this one or the one from terrelsa13?

**Comment by terrelsa13** [comment]
Hey @keppo070 

TLDR; They will both work. But the terrelsa13 repo has the latest updates.

Initially I was helping @clara-j update and maintain this script. I would push my changes into this trunk and it would be the most up to date. However @clara-j has mentioned they have higher priority things happening in life and have not been able to merge any of my changes.

To keep the project going I continue to maintain and update my repo [https://github.com/terrelsa13/media_cleaner](https://github.com/terrelsa13/media_cleaner).

Hope this helps. Also, if you have an issue/suggestion/request/etc... it will post here on @clara-j 's repo. Please @terrelsa13 so I receive an email and you don't have to wait 2 days or longer for me to manually check here for new issues.
